crates/forge/test_utils/src/lib.rs

```diff
@@ -1,3 +1,12 @@
-pub mod corelib;
 pub mod runner;
 pub mod running_tests;
+
+use anyhow::Result;
+use assert_fs::fixture::PathCopy;
+
+pub fn tempdir_with_tool_versions() -> Result<assert_fs::TempDir> {
```

Comment:
Might not be the place for it, will gladly take better ideas

Reply:
Let's put it somewhere in the `tests` dir

Reply:
Kept it here, as it is used in `forge/src/scarb.rs` tests and `test_utils` is a dev-dependency anyway